packages/react/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -726,7 +728,7 @@ MultiSelect.propTypes = {
    * Provide a compare function that is used to determine the ordering of
    * options. See 'sortItems' for more control.
    */
-  compareItems: PropTypes.func.isRequired,
+  compareItems: PropTypes.func,
```

Comment:
This is an optional prop to override the `defaultCompareItems` behavior. Should not be required.